src/components/postDetail/ImageModal/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { MouseEventHandler } from 'react';
+
+import classNames from 'classnames/bind';
+import ReactModal from 'react-modal';
+
+import styles from './ImageModal.module.scss';
+
+const cx = classNames.bind(styles);
+
+type ImageModalProps = {
+  isOpen: boolean;
+  imageSrc: string;
+  onClose: MouseEventHandler<HTMLButtonElement>;
+};
+
+const ImageModal = ({ isOpen, imageSrc, onClose }: ImageModalProps) => {
+  return (
+    <ReactModal
+      isOpen={isOpen}
+      onRequestClose={onClose}
+      shouldCloseOnEsc={true}
+      className={cx('image-modal')}
+      overlayClassName={cx('overlay')}
+      contentLabel='image-modal'
+    >
+      <div className={cx('image-modal-img')}>
```

Comment:
body-open 추가해야지 뒷배경 스크롤 이벤트 막을 수 있을 것 같습니다~

```suggestion
  return (
    <ReactModal
      isOpen={isOpen}
      onRequestClose={onClose}
      shouldCloseOnEsc={true}
      className={cx('image-modal')}
      overlayClassName={cx('overlay')}
      bodyOpenClassName={cx('body-open')}
      contentLabel='image-modal'
    >
      <div className={cx('image-modal-img')}>
```